app/src/swap/handle_sign_transaction.c

```diff
@@ -93,6 +129,10 @@ parser_error_t check_swap_conditions(parser_context_t *ctx_parsed_tx) {
     char tmpKey[20] = {0};
     char tmpValue[65] = {0};
 
+    if ((app_mode_expert() && ctx_parsed_tx->tx_obj->tx_json.num_items > 10) || (!app_mode_expert() && ctx_parsed_tx->tx_obj->tx_json.num_items > 6)) {
```

Comment:
Can we convert these numbers into defines?

Reply:
What if we have less? shouldn't it be equal signs?

Reply:
Good ideas!